content/docs/2.6/scalers/aws-dynamodb.md

```diff
@@ -0,0 +1,161 @@
++++
+title = "AWS DynamoDB"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on the records count in AWS DynamoDB"
+layout = "aws_dynamodb_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the AWS DynamoDB scaler. This scaler uses a specified DynamoDB query to determine if and when to scale a given workload.
+
+```yaml
+triggers:
+- type: aws-dynamodb
+  metadata:
+    # Required: awsRegion
+    awsRegion: "eu-west-1"
+    # Required: tableName
+    tableName: tableName
```

Comment:
Is table name the same as the resource name or?

Reply:
The tableName and region identify univocally the dynamoDB endpoint to retrieve the data. Therefore, I should have chosen a better value for `tableName`, such as `myTableName`. I'm proceeding by updating that in this PR.